.gitignore

```diff
@@ -16,6 +16,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+src/config/env/.env.dev
```

Comment:
You don't need to put explicit path here, just file name should be fine. :wink: 

And I would use config files that are already here e.g. `.env.development.local` rather then creating new ones. :sweat_smile:

Reply:
Ok :+1: :wink: